### PR TITLE
Add warnings on functions that don't exist in the database

### PR DESCRIPTION
**Pull request by angrykoala**
Similar to non-existing labels, a function that does not exist in the database will show as a warning

### CODE REVIEW

**Review by ncordon — CHANGES_REQUESTED** [review]
I think we are missing some edge cases:

* A test where there are spaces in between the dots `.`:

```
RETURN apoc    .     text     .    decapitalize    ("Marvin")
```


* A test to check the positions are calculated correctly when the function name is a escaped sequence:

```
RETURN     `some
multiline
thing`("Marvin")
```

* A test to check that this is still valid (because you can single escape individual parts of the functions):


```
RETURN `apoc`.coll.`avg`([1,2,3])
```

But this would get flagged as the function doesn't exist: ``RETURN `apoc.coll.avg`([1,2,3])``

* A test to check **only builtin** functions are case insensitive:

```
RETURN ABS(6.4)
```

So many edge cases we haven't tested or made the code work for yet

File: packages/language-support/src/parserWrapper.ts [hunk 1]
```diff
@@ -267,6 +287,79 @@ class VariableCollector implements ParseTreeListener {
...
+        column: ctx.start.column,
+        offsets: {
+          start: ctx.start.start,
+          end: ctx.stop.stop + 1,
+        },
+      });
+    }
+  }
+
+  private getFunctionName(ctx: FunctionNameContext): string {
```
Comment (ncordon) [inline packages/language-support/src/parserWrapper.ts:337]: Suggestion for making the code a bit more understandable: rename this to `getNormalizedFunctionName`

File: packages/language-support/src/parserWrapper.ts [hunk 2]
```diff
@@ -92,6 +95,17 @@ export type LabelOrRelType = {
   };
 };
 
+export type ParsedFunction = {
+  name: string;
+  rawText: string;
```
Comment (ncordon) [inline packages/language-support/src/parserWrapper.ts:100]: Maybe rename to `normalizedName` and `parsedName`?

File: packages/language-support/src/parserWrapper.ts (hunk 1 above)
Comment (ncordon) [inline packages/language-support/src/parserWrapper.ts:337]: Rename to `getNormalizedFunctionName` please

**Review by ncordon — APPROVED** [review]
